v2/Instrument Communications/Postman/InstrumentManagerWebApi.postman_collection.json

```diff
@@ -714,7 +714,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n\t\"createdBy\": \"Test Environment\",\n\t\"instrumentCalibration\": {\n\t\t\"laserWavelength\": 785.1547,\n\t\t\"arguments\": [7.874371e+02, 1.414304e-01, 1.095978e-05, -1.067649e-08, 1.853184e-12, -2.023975e-16],\n\t\t\"intensityCorrection\": {{IntensityCorrectionData}}\n\t}\n}"
```

Comment:
Three things here:

1. timestamp can be set with a built in variable
```
{{$isoTimestamp}}
```
3. spectrometer and unit serial numbers can also come from variables set in the Get Instrument call 
```
"timestamp": "{{$isoTimestamp}}",
"spectrometerSerialNumber": "{{SpectrometerSerialNumber}}",
"unitSerialNumber": {{UnitSerialNumber}}
```
4. The serial number variables would be set in the tests for Get Instrument like so
```
pm.environment.set("SpectrometerSerialNumber", pm.response.json().serialNumber);
pm.environment.set("UnitSerialNumber", pm.response.json().spectrometerSerialNumber);
```

Reply:
Added this code to use variables.

Defaults to emulator values for serial numbers, GetInstrument will set if needed.